cluster/examples/storage/gcp/resource-claim.yaml

```diff
@@ -0,0 +1,9 @@
+# Example RedisCluster resource claim using the cloud-rediscluster resource class
+apiVersion: storage.crossplane.io/v1alpha1
+kind: Bucket
```

Comment:
do we have more work to do to enable dynamic provisioning of buckets for Azure too?  Is that being tracked already?

Reply:
Nitpick: the comment on this example discusses RedisClusters.